<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>611</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Zone checker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../Users/pbodet/Downloads/saunier_logo.jpg</normaloff>../../../../../Users/pbodet/Downloads/saunier_logo.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="4" colspan="3">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="6" column="7">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="5" colspan="2">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Preview</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5" colspan="2">
       <widget class="QLabel" name="tc_number_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Raw image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="13">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="5" rowspan="2" colspan="2">
       <widget class="QLabel" name="input_view">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Zones</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="5" rowspan="2" colspan="2">
       <widget class="QLabel" name="output_view">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="4" colspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="4" rowspan="2" colspan="4">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="width_label">
           <property name="text">
            <string>width</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="5">
          <widget class="QSpinBox" name="height_spinbox">
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="5">
          <widget class="QSpinBox" name="y_spinbox">
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="x_label">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="y_label">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="width_spinbox">
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="x_spinbox">
           <property name="accelerated">
            <bool>true</bool>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="height_label">
           <property name="text">
            <string>height</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="3" rowspan="10">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="5">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" rowspan="10" colspan="2">
       <widget class="QListWidget" name="zones_list_widget">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="11" column="1" rowspan="2" colspan="7">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenConfig"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_copy_to_clipboard"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open Image...</string>
   </property>
  </action>
  <action name="action_copy_to_clipboard">
   <property name="text">
    <string>Copy to clipboard</string>
   </property>
  </action>
  <action name="actionOpenConfig">
   <property name="text">
    <string>Open Config...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
